### ISSUE TITLE
iOS App - On-Screen Keyboard covers text

### ISSUE THREAD

**Issue by esteve88** [hoarder-app/hoarder#795]
### Describe the Bug

On iOS application, when I'm opening a note which is quite long (like some which are also available in the demo app) and when I click on the content to enable the "edit" mode, I cannot edit the last items of the note (or even adding new lines at the bottom of the note) because the on screen keyboard appears and covers the remaining part of the note. I would have expected that the text of the note would have shifted up above the keyboard, like it happens for all other mobile apps.

### Steps to Reproduce

1. In iOS open a note which is quite long
2. Click on the note to enter edit mode
3. Try to edit one of the last lines or simply try adding a new line at the bottom of the screen - this will not work because the on screen keyboard appears and it will cover the bottom part of the note

### Expected Behaviour

All text of the note is visible above the keyboard

### Screenshots or Additional Context

![image](https://github.com/user-attachments/assets/9b773c36-8c78-4624-b1f1-bbe2c34a3783)

### Device Details

iOS 18.1.1

### Exact Hoarder Version

v0.20.0

### Have you checked the troubleshooting guide?

- [X] I have checked the troubleshooting guide and I haven't found a solution to my problem

**Comment by MohamedBassem** [comment]
Yeah, that's a bug. I fixed it in other places but forgot to fix it here.

**Comment by majant0** [comment]
Seems very similar to #266
